WordPress/Classes/ViewRelated/Reader/ReaderSearchViewController.swift

```diff
@@ -156,14 +156,15 @@ import Gridicons
     }
 
 
-    func configureLabel() {
+    @objc func configureLabel() {
         let text = NSLocalizedString("What would you like to find?", comment: "A short message that is a call to action for the Reader's Search feature.")
-        let attributes = WPNUXUtility.titleAttributes(with: WPStyleGuide.greyDarken20()) as! [String: AnyObject]
-        label.attributedText = NSAttributedString(string: text, attributes: attributes)
+        let rawAttributes = WPNUXUtility.titleAttributes(with: WPStyleGuide.greyDarken20()) as! [String: Any]
+        let swiftedAttributes = NSAttributedStringKey.convertFromRaw(attributes: rawAttributes)
```

Comment:
I see quite a few instances of "swifted" to refer to `NSAttributedStringKey` keyed attributes, and I don't think it's accurate. I haven't spent as much time with this as you, but from what I understand it's not about Swift vs Obj-C, but Foundation vs UIKit. I'd rather use `attributes`/`rawAttributes` everywhere and drop any "swifted" references.